frontend/src/features/login/LoginPage.tsx

```diff
@@ -175,7 +175,7 @@ export const LoginPage = (): JSX.Element => {
   }
 
   return (
-    <Flex flexDir="column" minH="100vh">
+    <Flex flexDir="column" h="100%">
```

Comment:
Sorry, some UI noob question here: Is `h` the same as `height` in the Flex component? When do we use one over the other?

Reply:
yes. they are just all aliases

Reply:
Right I was asking because just above in this PR, we use `height` (in `PublicElement.tsx`) but with a different unit `100vh`, so I was wondering if there are subtle differences 😅